codegen/pom.xml

```diff
@@ -72,7 +72,7 @@
         </dependency>
         <dependency>
             <groupId>org.mockito</groupId>
-            <artifactId>mockito-core</artifactId>
+            <artifactId>mockito-inline</artifactId>
```

Comment:
is there any reason we are using `mockito-inline` instead of `mockito-core`? I'd be surprised if we are using any inline mock features (mocking finals) in our tests.  Also, from the description of mockito-inline, it seems the artifact could be deprecated in the future.

>Mockito preconfigured inline mock maker (intermediate and to be superseeded by automatic usage in a future version)